packages/nuxt/src/core/runtime/nitro/renderer.ts

```diff
@@ -58,10 +58,18 @@ export interface NuxtIslandContext {
   name: string
   props?: Record<string, any>
   url?: string
-  // chunks to load components
-  chunks: Record<string, string>
-  // props to be sent back
-  propsData: Record<string, any>
+  slots: Record<string, Omit<NuxtIslandSlotResponse, 'html' | 'fallback'>>
+  clients: Record<string, Omit<NuxtIslandClientResponse, 'html'>>
```

Comment:
I'm not 100% sure about `clients` - can you think of another word?

Reply:
Do you have any suggestion ?

Reply:
only having a minimal knowledge of how this is all working, clients hints at a list of socket connections or something

Reply:
Maybe selectiveClient or simply `components`?

Reply:
`components` sounds good to me then, it's clear enough that non-hydrated HTML won't have a component

Reply:
in that case, should we also rename data-island-client to data-island-component ?

Reply:
oops, I missed this 🤦 

opened https://github.com/nuxt/nuxt/pull/25232